### ISSUE TITLE
Feature request: `thebibliography` environment

### ISSUE THREAD

**Issue by odomanov** [plk/biblatex#783]
I'm not sure if you can see this one: https://github.com/plk/biblatex/issues/292#issuecomment-414399008, so I opened a new ticket. Could you please comment whether this approach can be improved and somehow incorporated into Biblatex?

**Comment by odomanov** [comment]
Thank you for the lengthy comment. 
Sorry, that thread was closed so I was not sure it is visible.)))

I expected of course that this is not easy. I just thought that this is a nice workaround which solves quite an annoying shortcoming of Biblatex (although I've personally never run into it since I happened not to submit to such journals). Of course the problem is journal submissions --- those when you don't use `.bst` files but need to supply `thebibliography` environment directly. `bst` templates do not really help there and it would be nice to use `biblatex` for formatting.

Actually I just like the @nickkolok's idea --- to use Biblatex itself to form what is needed. The only problem is how to make the output not to pdf but to a file. I don't know TeX internals so I thought this might be possible on some "deep" level. Perhaps `luatex` can do this. Of course we need to decide about the "level of expansion" but I don't think we need too much macros -- just `mkbibemph`, `mkbibbold`, maybe a couple else. You might be right, this is not quite a `biblatex` task and can be better done with `.bst`. But there are things `bibtex` is not very good with, for example multilingual bibliographies (I need Biblatex mostly for them; and I actually started using TeX extensively only after `biblatex` appeared). Well, OK, I'm not sure.

Anyway, the method is useful, I think, although limited, and may be used as is. I hope someone can improve it.

**Comment by josephwright** [comment]
@moewew Totally agree: there's a reason PL didn't include this at the start. The entire model is different from classical BibTeX, and although journal submission is an obvious area of issue, really trying to 'fix' it by munging `biblatex` is not a great plan.

**Comment by nickkolok** [comment]
Uploaded a package to CTAN: https://ctan.org/pkg/biblatex2bibitem
GitLab repo: https://gitlab.com/Nickkolok/biblatex2bibitem

**Comment by moewew** [comment]
In order to avoid two concurrent threads with the same contents. I will lock this one here, so we can focus discussion on #292.
